docs/tutorial.rst

```diff
@@ -0,0 +1,1613 @@
+
+.. code-block:: shell
+
+    (psamm-env) $ psamm-import ecoli_textbook --source E_coli_excel/ecoli_core_model.xls --dest converted_excel_model
+
+This will produce a yaml version of the excel model in the
+converted_excel_model/ directory.
+
+Since the EXCEL models are not in a standardized format these parsers need to
+be developed on a model by model basis in order to parse all of the relevant
+information out of the model. Future support will be added for more EXCEL
+based models as the parsers are developed.
+
+D) Importing a JSON Model
+
+``psamm-import`` also supports the conversion of JSON format models that follows
+the conventions in COBRApy. If the JSON model is stored locally, it can be
+converted with the following command:
+
+.. code-block:: shell
+
+    (psamm-env) $ psamm-import cobra-json --source e_coli_json/e_coli_core.json --dest converted_json_model/
+
+Alternatively, an extension of the JSON importer has been provided,
+``psamm-import-bigg``, which can be applied to convert JSON models from `BiGG`_
+database .  To see the list of available models on the
+BIGG database the following command can be used:
+
+.. code-block:: shell
+
+    (psamm-env) $ psamm-import-bigg list
+
+This will show the available models as well as their names. The user can then
+import any of these models to YAML format. For example, using the following
+command to import the E. coli iJO1366 model form the bigg database:
+
+.. code-block:: shell
+
+    (psamm-env) $ psamm-import-bigg iJO1366 --dest converted_json_model_bigg/
+
+.. note::
+    To use ``psamm-import-bigg`` the user must have internet access so
+    that the models can be downloaded by the program.
+
+4) YAML Format and Model Organization
+______________________________________
+
+The `PSAMM` YAML format stores individual models under a designated directory,
+in which there will be a number of files that stores the information of the
+model and specifies the simulation conditions. The entry point of the YAML
+model is a file named model.yaml, which points to additional files that store
+the information of the model components, including compounds, reactions, flux
+limits, medium conditions, etc. While we request the users to use the name
+‘model.yaml’ for the central reference file, the file names for the included
+files are flexible and can be customized by the users. In this tutorial, we
+simply used the names: compounds.yaml, reactions.yaml, limits.yaml, and
+medium.yaml for the include files, respectively. To view these files directly
+the user can first change directory into the E_coli_yaml file using the
+following command:
+
+.. code-block:: shell
+
+    (psamm-env) $ cd E_coli_yaml/
+
+The central file in this organization is the model.yaml file. The following
+is an example of the model.yaml file that is obtained from the import of the
+E. coli textbook model. The model.yaml file for this imported sbml model should
+look like the following:
```

Comment:
Perhaps putting the code to print the contents of the file ex:  (psamm-env)~/psamm-tutorial/E_coli_yaml  $ cat model.yaml